src/pybind/mgr/nfs/export.py

```diff
@@ -44,6 +46,16 @@
 
 log = logging.getLogger(__name__)
 
+cephfs = None   # holds CephFS Python bindings
+
+
+def is_path_valid(path, str_type=b''):
+    try:
+        return (cephfs.stat(os.path.join(str_type, path)).st_mode
```

Comment:
Doesn't this require a client initialization phase (cephfs.mount())?

Reply:
That would only be needed during testing i.e. in qa tests. cephfs from `cephfs.pyx` would be able to find the filesystem mounted and run `stat()`.

Reply:
this could be better placed in `cephfs.pyx`, it does not contain any function to check if the cephfs path is valid or not. I can also see similar function in cephfs-shell to check for path that uses `cephfs.pyx`(a libcephfs wrapper), creating same function everywhere would lead to redundancy

Reply:
Sounds fine.

Reply:
added, PTAL

Reply:
as we need a fs connection handle to carry out `stat`, using `cephfs.pyx` directly is not the right way, therefore removed the functions from `cephfs.pyx`, closing the conversation.